src/app/pages/SearchPage/searchPage.module.css

```diff
@@ -0,0 +1,20 @@
+.container {
+  display: grid;
+  grid-template-rows: auto auto;
+  justify-items: center;
+}
+
+.footer {
+  display: grid;
+  grid-template-columns: repeat(3, 33%);
```

Comment:
I think ou could write `grid-template-column: repeat(3, 1fr)`. This would make the columns exactly one third each.